Enable create-spdx when building images.

dylan-matthew-turner opened the pull request:
Because [we now skip creating SPDX when a package comes from a remote feed](https://github.com/ni/openembedded-core/pull/90), we can enable creating the spdx zst archives when building images.

To enable it, we need to inherit from create-spdx at the top-level bitbake config, and add it to the list of DISTRO_FEATURES.

I verified this produces the expected archive by building `packagefeed-ni-core` followed by `nilrt-base-system-image`

SparkingSpork commented on conf/distro/nilrt.inc:
```diff
@@ -19,15 +19,16 @@ DISTRO_FEATURES += "\
 
 # Clean *most* package work directories
-INHERIT += " rm_work "
+INHERIT += " rm_work create-spdx "
```
Can you add the `create-spdx` tag in a separate `INHERIT +=` line for clarity? This section is labeled as for configuring how the build handles temporary workspaces for each package. We probably want a separate section with comment that adds just the `create-spdx` tag.

amstewart replied:
MINOR: since the spdx artifacts are a nuance of how _we_ (NI) would like to build the distribution, instead of being a fundamental part of NILRT, I think this code should be moved to `nilrt.conf`.

dylan-matthew-turner committed:
nilrt.inc: tab to spaces for DISTRO_FEATURES

Under DISTRO_FEATURES, all features listed used spaces not tabs except
virtualization. Make it consistent with the rest of the formatting

Signed-off-by: Dylan Turner <dylan.turner@ni.com>

dylan-matthew-turner committed:
nilrt.conf: enable spdx creation

Enable create-spdx by inheriting the class and set it to run on image
creation by adding it to the distro feature set

Signed-off-by: Dylan Turner <dylan.turner@ni.com>

dylan-matthew-turner commented:
I moved the changes to nilrt.conf from nilrt.inc (and thus also split the INHERIT), but I left a change in nilrt.inc to convert one tab in a string of spaces to all spaces for the sake of consistency.

I rebuilt after moving, and the spdx is still being created
